commit 75f395df8510bfad2da779edcedeb5a1ec3ab63f (refs/changes/72/3672/3)
Author: Brian Bennett <brian.bennett@joyent.com>
Date:   2018-03-16T18:20:04-07:00 (1 year, 7 months ago)
    
    MANATEE-398 manatee-adm history -j doesn't understand v1 timestamps
    Reviewed by: David Pacheco <dap@joyent.com>

diff --git a/lib/adm.js b/lib/adm.js
index 411ef5d..92b2636 100644
--- a/lib/adm.js
+++ b/lib/adm.js
@@ -1814,8 +1814,7 @@ function oldHistoryToObj(fNode) {
                      fNode[j] === 'null') ?  '' : fNode[j];
         switch (j) {
         case 0:
-            node.time = entry;
-            node.date = new Date(parseInt(entry, 10));
+            node.time = new Date(parseInt(entry, 10));
             break;
         case 1:
             node.ip = entry;
